contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: "BitcoinUnlimited-win-0.12"
+name: "BitcoinUnlimited-win-1.0"
```

Comment:
I do have to agree with @deadalnix here. It detracts from the code that we should be focusing on.